### ISSUE TITLE
"screen -c .screenrc" should be launched at startup

### ISSUE THREAD

**Issue by kartben** [kartben/eclipseiot-demobox#3]

**Comment by mmilinkov** [comment]
If you want to start this automatically at startup, is screen the right way to go? Perhaps either a service, or even a simply cron job would be easier?

**Comment by kartben** [comment]
Good point! However I think screen still is quite convenient since it provides an easy and central way to see the logs/console outputs of all the various pieces of the demo in one unique window. 

I think at the end of the day I will end up wrapping the call to "screen -c .screenrc" in a service, indeed.
